.github/workflows/backup-media.yml

```diff
@@ -0,0 +1,24 @@
+name: Backup media
+
+concurrency: stage
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '50 2 * * MON' # At 02:50 UTC every Monday
+
+jobs:
+  backup-media:
+    environment: StagingRO
+    runs-on: [self-hosted, stage]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: media_backup_src
+      - name: Run backup
+        env:
+          MYSQL_ROOT_PASSWORD: root
```

Comment:
Can we replace it with an environment secret?

Reply:
Sure, but it's not very secret :)